MediaPlayer/MediaPlayer.module.less

```diff
@@ -2,6 +2,16 @@
 
 .mediaPlayer {
 	.applySkins({
+		background-color: @agate-mediaplayer-bg-color;
```

Comment:
I don't think GUI wants to put this color as the background color.
I think she put it to show the area that the player takes in.
Please see https://app.zeplin.io/project/5fab9d1c39e44d926043ec54/screen/603dedd0903bd2712fda1c78.

Reply:
I got tricked :)